packages/nimble-components/src/anchor-menu-item/template.ts

```diff
@@ -0,0 +1,39 @@
+import { html, ref, ViewTemplate } from '@microsoft/fast-element';
+import {
+    AnchorOptions,
+    FoundationElementTemplate,
+    startSlotTemplate
+} from '@microsoft/fast-foundation';
+import type { AnchorMenuItem } from '.';
+
+export const template: FoundationElementTemplate<
+ViewTemplate<AnchorMenuItem>,
+AnchorOptions
+> = (context, definition) => html<AnchorMenuItem>`
+    <template
+        role="menuitem"
+        class="${x => (typeof x.startColumnCount === 'number'
+        ? `indent-${x.startColumnCount}`
+        : '')}"
+        aria-disabled="${x => x.disabled}"
+    >
+        <a
+            ${ref('anchor')}
+            download="${x => x.download}"
+            href=${x => (x.disabled ? null : x.href)}
+            hreflang="${x => x.hreflang}"
+            ping="${x => x.ping}"
+            referrerpolicy="${x => x.referrerpolicy}"
+            rel="${x => x.rel}"
+            target="${x => x.target}"
+            type="${x => x.type}"
+            @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
+            @keydown="${(x, c) => x.keydownHandler(c.event as KeyboardEvent)}"
+        >
+            ${startSlotTemplate(context, definition)}
```

Comment:
The usual question about start and end slots, for example: https://github.com/ni/nimble/pull/832#discussion_r1041475584

> FastAnchor and AnchorOptions include the start and end properties. We should include them in the template and hide them in styles or make sure our types are cleaned correctly.

Seems like there is some gap as this keeps being discussed in review. Maybe there should be re-usable tests for verifying the start and end properties are defined and functioning as expected (either slotting values or display none)>

Reply:
Yeah, I keep forgetting about those. I've now applied the mixin to the class, and I've added two tests to make sure those properties get the slotted content and that they are being displayed (or not) as expected. I wasn't sure how to make reusable tests, so for now they are not.